terraform_modules/base_framework/providers.tf

```diff
@@ -0,0 +1,56 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+    local = {
+      source  = "hashicorp/local"
+      version = ">= 2.1"
+    }
+    tls = {
+      source  = "hashicorp/tls"
+      version = ">= 3.1"
+    }
+    cloudinit = {
+      source = "hashicorp/cloudinit"
+      version = "2.3.2"
+    }
+  }
+
+  backend s3 {
+    bucket         = "nr-coreint-canaries-tfstates"
+    dynamodb_table = "nr-coreint-canaries-tflocking"
+    key            = "base-framework/global-state-store.tfstate"
```

Comment:
I have the feeling that this state key will become misleading in the near future...

Could we find a better name? I think something like `coreint-canaries/foundations/terraform-states-backend.tfstate` would be better.

Reply:
👍